android/LocalNotifyPlugin.java

```diff
@@ -410,13 +413,8 @@ public void onCreate(Activity activity, Bundle bundle) {
 			logger.log("{localNotify} Initializing");
 
 			_launchName = null;
+			onNewIntent(activity.getIntent());
```

Comment:
doent looks like this function call have any effect, since onNewIntent action check fails always since action here is "android.intent.action.BOOT_COMPLETED"